builder_requests.go

```diff
@@ -135,25 +135,25 @@ func (c *Client) BuildFunction(module *Module, template string, functionBody io.
 	return buildResult, nil
 }
 
-// BuildFunctionString triggers a remote build for the given Module and function string. See also: Client.BuildFunction()
-func (c *Client) BuildFunctionString(module *Module, template, functionString string) (*BuildResult, error) {
-	buf := bytes.NewBufferString(functionString)
-	return c.BuildFunction(module, template, buf)
+// BuildPluginString triggers a remote build for the given Plugin and source code. See also: Client.BuildPlugin()
+func (c *Client) BuildPluginString(plugin *Plugin, template, source string) (*BuildResult, error) {
+	buf := bytes.NewBufferString(source)
+	return c.BuildPlugin(plugin, template, buf)
 }
 
-// GetDraft gets the most recently build source code for the provided Module. Must have the .FQFNURI field set.
-func (c *Client) GetDraft(module *Module) (*EditorStateResponse, error) {
-	if module == nil {
-		return nil, errors.New("Module cannot be nil")
+// GetDraft gets the most recently build source code for the provided Plugin. Must have the .FQFNURI field set.
```

Comment:
```suggestion
// GetDraft gets the most recently build source code for the provided plugin. Must have the .FQMNURI field set.
```
Nit based on feedback in other repos.